src/SEnergyCorrelatorConfig.h

```diff
@@ -49,6 +49,10 @@ namespace SColdQcdCorrelatorAnalysis {
     float pTSmear = 0;
     float theta = 0;
 
+    //Manual Calculation options
+    bool manualTwoPoint = false;
+    bool manualThreePoint = false;
+
```

Comment:
Following above comments. I would also suggest placing this just after the `doPackageCalc`.

```suggestion
    bool doManualCalc = false;

```